pkg/ranger/content.go

```diff
@@ -95,7 +95,7 @@ func ServeContent(w http.ResponseWriter, r *http.Request, name string,
 					amount = sniffLen
 				}
 				// TODO: cache this somewhere so we don't have to pull it out again
-				r := content.Range(0, amount)
+				r, _ := content.Range(0, amount)
```

Comment:
if err != nil, r might be nil. you still have to check the error, even if it means we don't set ctype.